collector/dhcp_collector.go

```diff
@@ -34,9 +36,9 @@ type dhcpCollector struct {
 }
 
 type dhcpStatusMetric struct {
-	ID     string
-	Name   string
-	Status float64
+	ID           string
+	Name         string
+	StatusDetail map[string]float64
```

Comment:
can we use enum of states here instead of string? it will help guarding the values

Reply:
I'm not sure how to achieve this, can you share some examples?